accounts-db/src/accounts_hash.rs

```diff
@@ -61,18 +61,97 @@ impl MmapAccountHashesFile {
 
 /// 1 file containing account hashes sorted by pubkey
 struct AccountHashesFile {
-    /// # hashes and an open file that will be deleted on drop. None if there are zero hashes to represent, and thus, no file.
+    /// The mmap hash file created in the temp directory, which will be deleted on drop.
     writer: Option<MmapAccountHashesFile>,
-    /// The directory where temporary cache files are put
```

Comment:
The following fields are only needed for "creating the hash mmap file". 
Now we don't need them since the mmapfile has already created before write.